tests/js/sentry-test/queryClient.tsx

```diff
@@ -0,0 +1,26 @@
+import {QueryClient} from '@tanstack/react-query';
+import merge from 'lodash/merge';
+
+import {DEFAULT_QUERY_CLIENT_CONFIG} from 'sentry/utils/queryClient';
+
+export const makeTestQueryClient = () =>
```

Comment:
Moved this to its own file and exported it so it can be imported in test files (I needed to do this to wrap the hook test in a QueryClientProvider)